apps/console/src/features/applications/components/forms/inbound-oidc-form.tsx

```diff
@@ -814,6 +814,20 @@ export const InboundOIDCForm: FunctionComponent<InboundOIDCFormPropsInterface> =
                         readOnly={ readOnly }
                         data-testid={ `${ testId }-access-token-type-radio-group` }
                     />
+                    <Message compact={false} size={"tiny"} className={"border-less"}>
+                        <p>
+                            Asgardeo has the capability to attach the OAuth2 access token and refresh
```

Comment:
In the product "WSO2 Identity Server", and these should come from i18n file.

Reply:
We need to take i18n to extensions as well. So we can override specific messages like these. Will create an issue for this